src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt

```diff
@@ -57,4 +57,29 @@ class UserController(
     username: String
   ) = userService.findUserByUsername(username)
     ?: throw NotFoundException("Account for username $username not found")
+
+  @GetMapping("/user/me/roles")
```

Comment:
Happy either way but should we use /users rather than /user to match the other endpoints?

Reply:
updated to users